### PR TITLE
type mismatche

### PR DESCRIPTION
**Pull request by Bolukan**
missed ‘s’ in espreSsIf
use possible entries based on https://docs.platformio.org/en/latest/platforms/index.html#platforms

### CODE REVIEW

**Comment by Bolukan** [comment]
PS: I tested the current release: as both 'espressif' and 'espresif8266' don't exist, it loads the library for all platforms: compiling for ESP32 returns errors. 
If at least one platform is defined (e.g. 'espressif8266') the other ('espressif') is ignored, and the code compiles well for the ESP32

**Comment by gmag11** [comment]
Let me check Platform.io documentation. I do not use it and cannot test it so often. I'll try to find the correct esp32 descriptor

File: library.json [hunk 1]
```diff
@@ -14,7 +14,7 @@
     {
       "name": "ESPAsyncUDP",
       "authors": "Hristo Gochkov",
-      "platforms": ["espressif", "espresif8266"]
```
Comment (gmag11) [inline library.json:17]: This should be `"platforms": ["espressif32", "espresif8266"]` as it is listed on https://docs.platformio.org/en/latest/platforms/espressif32.html

May you test if it compiles ok?

**Comment by Bolukan** [comment]
The library EspAsyncUDP is not suited for the Esp32, but only for the Esp8266.

For the Esp32 AsyncUDP was developed by the same author, but this library became part of the arduino esp32 framework in June 2018 and does not need to be downloaded seperately anymore.

If you use “espressif8266” instead of “espresif8266” (double s instead of single s) and remove reference to espressif32 in library.json it compiles well on the esp32.

I tested this for esp32.

File: library.json (hunk 1 above)
Comment (Bolukan) [reply library.json:17]: No it compiles not ok. see thread for comment.

**Pull request merged**